packages/flutter/lib/src/widgets/dismissible.dart

```diff
@@ -503,11 +573,41 @@ class _DismissibleState extends State<Dismissible> with TickerProviderStateMixin
       return;
     }
     _dragUnderway = false;
+
+    final double flingVelocity = _directionIsXAxis ? details.velocity.pixelsPerSecond.dx : details.velocity.pixelsPerSecond.dy;
+
+    // Use value returned by `shouldTriggerDismiss` if a callback is provided
+    // If the callback returns null, use the default behavior
```

Comment:
```suggestion
    // Use value returned by `shouldTriggerDismiss` if a callback is provided.
    // If the callback returns null, use the default behavior.
```

One [style guideline](https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#use-correct-grammar) is "end all sentences with a period."